frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/FeatureOverviewEnvironment.tsx

```diff
@@ -179,6 +185,11 @@ const FeatureOverviewEnvironment = ({
                                         environmentId={env.name}
                                         variant='outlined'
                                         size='small'
+                                        disableReason={
+                                            limitReached
+                                                ? 'Limit reached'
```

Comment:
For the reasons, can we be more explicit? Just "limit reached" doesn't tell me what limit or what the limit is. Maybe something like "you can't add more than 30 strategies to this environment" or something? 🤷🏼

Reply:
I will do this with the config change